comms/core/src/transports/memory.rs

```diff
@@ -137,7 +137,7 @@ mod test {
 
     #[tokio::test]
     async fn simple_listen_and_dial() -> Result<(), ::std::io::Error> {
-        let t = MemoryTransport::default();
+        let t = MemoryTransport;
```

Comment:
is this a new style / default? Feels foreign and anti-intuitive.

Reply:
Believe it is,